### ISSUE TITLE
Incorrect definition for min() and max()

### ISSUE THREAD

**Issue by xzyfer** [sass/libsass#826]
Libsass currently defines function signatures of `min()` and `max()` as follows.

```
min($x1, $x2...)
max($x1, $x2...)
```

However Ruby sass defines them as follows

```
min($numbers...);
max($numbers...);
```

This is related to https://github.com/sass/libsass/issues/813.

**Comment by xzyfer** [comment]
Spec added https://github.com/sass/sass-spec/pull/238.

**Comment by xzyfer** [comment]
This is fixed and will be 3.2.
